tests/unit/app/testQueryFunctionFactory.test.ts

```diff
@@ -0,0 +1,34 @@
+import { mocked } from 'ts-jest/utils';
+import queryFunctionFactory from '../../../src/app/testQueryFunctionFactory';
+import * as dbFunctions from '../../../src/app/databaseService';
+import DatabaseService from '../../../src/infrastructure/databaseService';
+
+jest.mock('../../../src/app/databaseService');
```

Comment:
Is the automocking reset when the tests finished executing?
If remember correctly when using spyOn, etc.. mockRestore or reset is required.

Reply:
These work slightly differently, so they don't need resetting, see https://stackoverflow.com/questions/56496998/how-to-restore-a-mock-created-with-jest-mock#comment112081343_56512217 (and the associated answer)

Reply:
thanks for sharing that 👍